core.coffee

```diff
@@ -230,7 +230,7 @@
 			apiPrefix: '/'
 			passthrough: {}
 			passthroughByMethod: {}
-			constructor: (params) ->
+			constructor: (params, backendParams) ->
```

Comment:
We always required instantiation of the backend class directly, but the backend class extends the core class (and most of the work should be done by the core class).

I'll add a comment